### PR TITLE
fix: fix grid overflow error

### PR DESCRIPTION
**Pull request by laojun**
## What this PR does / why we need it:
fix: fix grid overflow error

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    fix: fix grid overflow error          |
| 🇨🇳 中文    |   fix: 修复grid组件滑动问题           |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes #
https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=244068&issueFilter__urlQuery=eyJzdGF0ZXMiOls0NDAyLDcxMDQsNzEwNSw0NDAzLDQ0MDQsNzEwNiw0NDA2LDQ0MDcsNDQxMiw0NTM4LDQ0MTMsNDQxNCw0NDE1LDQ0MTZdLCJhc3NpZ25lZUlEcyI6WyIxMDM5MCJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=541&type=BUG

### CODE REVIEW

File: shell/app/config-page/components/grid/grid.tsx [hunk 1]
```diff
@@ -22,12 +22,14 @@ export default (props: CP_GRID.Props) => {
...
   return (
-    <Row gutter={gutter}>
-      {children.map((child, i) => (
-        <Col span={itemSpan[i]} key={i}>
-          {child}
-        </Col>
-      ))}
-    </Row>
+    <div className="overflow-hidden">
+      <Row gutter={gutter}>
```
Comment (daskyrk) [inline shell/app/config-page/components/grid/grid.tsx:26]: Why not add className to Row?

Comment (laojun) [reply shell/app/config-page/components/grid/grid.tsx:26]: it's useless add to row, suppose to be forbidden overflow out of Row

**Pull request merged**